Intro to Linux Tutorial

Sumaya112000 opened the pull request:
Removed links that are not working and changed host names to match current machines (the tutorial mentioned wheeler and galles).

Sumaya112000 committed:
Update tutorial links and instructions

Removed links that are not working and changed host names to match current machines (the tutorial mentioned wheeler and galles).

gmfricke reviewed (changes requested):
@Graviton28 Instead let's link to the software carpentries: [https://swcarpentry.github.io/shell-novice/01-intro.html)]( https://swcarpentry.github.io/shell-novice/01-intro.html). Add some wrapper text perhaps describing why the software carpentries are a good resource. Also link to the intro to CARC for beginners slides? They go over BASH basics.

Graviton28 committed:
Update linux_intro.md

Replace our tutorial with a referance to the Software Carpentry one, added some specific information about using our specific systems.
